cli/src/command.rs

```diff
@@ -605,6 +605,29 @@ pub fn run() -> Result<()> {
 				.into(),
 		)
 		.into()),
+		Some(Subcommand::ChainInfo(cmd)) => {
+			let runner = cli.create_runner(cmd)?;
+			let chain_spec = &runner.config().chain_spec;
+
+			ensure_dev(chain_spec).map_err(Error::Other)?;
```

Comment:
This would ensure that it only works on `*-dev` runtimes, not on main net.  
Did you try it out?

Reply:
I did, with dev.. let me fix it.